### DISCUSSION TITLE
Tone of old Mac OS notification

### DISCUSSION THREAD

**Discussion by Jmarinis** [Homebrew/discussions#4487] [Everyday usage]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.16
ORIGIN: https://github.com/Homebrew/brew
HEAD: 01e0c20d01c6aab28d844280d130f0e745e2ddd9
Last commit: 6 days ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: d91fe41310d84ca26fb3f2069b5225334c6cfee2
Core tap last commit: 15 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_INSTALL_FROM_API: set
Homebrew Ruby: 2.6.10 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.10_1/bin/ruby
CPU: quad-core 64-bit westmere
Clang: 10.0.0 build 1000
Git: 2.17.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.54.0 => /usr/bin/curl
macOS: 10.13.6-x86_64
CLT: 10.1.0.0.1.1539992718
Xcode: 10.1
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: You are using macOS 10.13.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.
```

### Description of issue

Thank you for all the work you do (and have done) to develop and maintain Homebrew.

In that spirit, perhaps you could consider a less hostile message when a person using an older Mac attempts to update their Homebrew environment.  Maybe a tone that provides understanding even.  It would be a trivial step and would do a much better job of promoting your work.

Thanks again.

**Comment by p-linnane** [comment]
This message was updated to this language in https://github.com/Homebrew/brew/commit/53b1c0f48e7befe5260f6dbf20aec32fb847caee. It's spelled out very bluntly because users who are on unsupposed macOS versions are on their own. We, as maintainers, only support the versions of macOS that Apple actively supports. Every fall when Apple drops support for a version of macOS, we do the same. We will not encourage or provide support for  remaining on unsupported and insecure OS versions.

**Comment by SMillerDev** [comment]
> Maybe a tone that provides understanding even. It would be a trivial step and would do a much better job of promoting your work.

We used to have that, it resulted in people not honouring those rules. Unfortunately that left us with no other choice.
